### ISSUE TITLE
Support for `tsconfig.json` Paths

### ISSUE THREAD

**Issue by GeisonJr** [mcollina/borp#117]
**Description**:
I'm using the `borp` package with a TypeScript project, and I noticed that `borp` doesn’t currently support paths defined in the `tsconfig.json` file. This feature would be highly beneficial for maintaining cleaner import paths and improving code organization.

**Suggested Solution**:
Add support for resolving module paths based on `tsconfig.json` configurations. This would allow for seamless integration of custom import paths, enabling shorter and more manageable imports across projects.

**Example**:
Here’s an example of the `tsconfig.json` paths section that `borp` would ideally support:

```json
{
  "compilerOptions": {
    "incremental": true,
    "baseUrl": "./",
    "paths": {
      "@components/*": ["src/components/*"],
      "@utils/*": ["src/utils/*"]
    }
  }
}
```

**Comment by mcollina** [comment]
Thanks for reporting! Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by GeisonJr** [comment]
Yes, I'm working on it. I'll prepare a Pull Request addressing this issue and include unit tests.
